lib/hobo_fields.rb

```diff
@@ -43,7 +47,7 @@ module HoboFields
   attr_reader :field_types
 
   def to_class(type)
-    if type.is_one_of?(Symbol, String)
+    if type.is_a?(Symbol) || type.is_a?(String)
```

Comment:
can't this be simplified to a simple case statement?
```
case type
when Symbol, String
  ...
else
  ...
end
```

Reply:
Sure, that's a good idea!

Reply:
Changed here: https://github.com/Invoca/hobo_fields/pull/16/commits/dfc1ce2a5b5af49bf03218e87a0d204fe7fe5b8a
No tests broke. :P